BD-3245: Canvas templates EA - MERGE 7/18 (#7682)
braze-inc/braze-docs: _docs/_hidden/private_betas/canvas_templates.md

---
nav_title: Canvas Templates
article_title: Canvas Templates
permalink: "/canvas_templates/"
description: "This reference article covers how to create a template for Canvas."
page_type: reference
---

# Creating a Canvas template

> This reference article covers how to create and manage templates for Canvas. Using templates can refine your messaging by creating a consistent framework that can be easily customized to fit your specific goals across your Canvases.

{% alert important %}
Canvas templates are currently in early access. Contact your customer success manager if you're interested in participating in this early access.
{% endalert %}

> ## Step 1: Navigate to the Canvas template editor

Reviewer: Is "Navigate" the most human phrase here? You use "Go" below. 

Honest, open question: Do we have a different feelings about the tone of headings vs prose?

Author: No strong feelings! I mirrored the heading language in a doc we have for [creating webhook templates](https://www.braze.com/docs/user_guide/message_building_by_channel/webhooks/webhook_template#step-1-navigate-to-the-webhook-template-editor)

In the Braze dashboard, go to **Templates** > **Canvas Templates**.

{% alert note %}
If you are using the older navigation, you can find this page under **Engagement** > **Templates & Media** > **Canvas Templates**.
{% endalert %}

## Step 2: Create a new template

> Select **Create template** to begin setting up your Canvas details. Give your Canvas template a name.

Reviewer: Curious about the tense here. "To begin setting up." I feel like you were drawing on something here. What are your thoughts?

Love "Give your Canvas template a name."

![An example Canvas template named "Annual sale Canvas template" with the description "Use for annual spring promotion".][1]

## Step 3: Customize your template

> Next, customize your template by building your Canvas as you normally would. This is where you can decide when users should enter the Canvas, determine which users can enter this Canvas, adjust your send settings, and build your user journey for the template.

Reviewer: "As you normally would" feels a little off to me because it doesn't feel like this section of the documentation is going to smoothly transfer back into normal documentation. Would we respect the user experience more by recasting this phrase to avoid saying "normally would," or should we provide a link to a guide to building Canvases?

## Step 4: Save your template

> After you've finished customizing your template, sure to save it by selecting the **Save template** button. On the **Canvas template** page, you can view your Canvas template details by selecting <i class="fas fa-list"></i> **Template details**. 

Reviewer: ```suggestion
After you've finished customizing your template, select the **Save template** button. On the **Canvas template** page, you can view your Canvas template details by selecting <i class="fas fa-list"></i> **Template details**. 
```

> ## Step 5: Use your template

Reviewer: Is this step 5 of "Creating", or is this a separate section?

Perhaps "## Using templates"

Author: good catch! i'll update the heading
